src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/SignatureContext.cs

```diff
@@ -187,7 +187,7 @@ private static bool MethodIsSkipLocalsInit(StubEnvironment env, IMethodSymbol me
             return false;
 
             static bool IsSkipLocalsInitAttribute(AttributeData a)
-                => a.AttributeClass?.ToDisplayString() == TypeNames.System_Runtime_CompilerServices_SkipLocalsInitAttribute_Metadata;
+                => a.AttributeClass?.ToDisplayString() == TypeNames.System_Runtime_CompilerServices_SkipLocalsInitAttribute;
```

Comment:
Does this check still work after refactoring -- comparing DisplayString of an INamedTypeSymbol to a NameSyntax?

Reply:
This is still a string on the right side. Changes to TypeNames were only renames. NameSyntaxes and TypeSyntaxes contain the static NameSyntax and TypeSyntax properties.